docs/02-hugo.Rmd

```diff
@@ -695,7 +695,7 @@ Lastly, this theme provided an example site under `exampleSite/`. The directory
 
 The example site provided a sample `config.toml`, a home page `_index.md`, an about page `about.md`, two posts under `note/` and two under `post/`. It also overrides the `foot_custom.html` in the theme.
 
-### How-to
+### How-to update
```

Comment:
Actually I don't like either `How-to` or `How-to update`. Other ideas? `Implementing More Features`?

Reply:
Yes! I like that. I'll update it.